enterprise/server/cmd/ci_runner/bundle/bundle.go

```diff
@@ -3,15 +3,19 @@ package bundle
 import (
 	"embed"
 	"io/fs"
-
-	"github.com/buildbuddy-io/buildbuddy/server/util/fileresolver"
 )
 
+const RunnerName = "buildbuddy_ci_runner"
+
 // NB: Include everything in bazel `embedsrcs` with `*`.
 //
-//go:embed *
+//go:embed buildbuddy_ci_runner
 var all embed.FS
```

Comment:
we can simplify by changing this to

`var RunnerBytes []byte`

instead of

`var all embed.FS`

Then we wouldn't need to deal with errors anywhere since the error check would effectively be moved to compile time. In `workspace.go` instead of copying from one file to another, we can do a single `os.WriteFile` call.

Reply:
It has been a while but I'm back!

could you please take another look?